src/esn.inbox.libs/app/directives/inbox-emailer-display/inbox-emailer-display.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+const _ = require('lodash');
+
+angular.module('esn.inbox.libs')
+
```

Comment:
```suggestion
```

Reply:
I think this is not necessary, we have many places in our code that have this convention. A blank line makes it easier for eyes